lib/presenter/Chart.js

```diff
@@ -205,12 +206,17 @@ class Chart extends Events {
         delete this.yRange;
         delete this.zoomX;
         delete this.zoomY;
+        delete this._guideLayerEl;
 
         this.appendTo(selectorOrEl);
+
+        this.trigger('afterResize');
```

Comment:
New event here (useful when you need to add custom d3 stuff after a resize like the "end lines" in the projection card)